texttospeech/ssml_addresses/tts_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tts import text_to_ssml
+from tts import ssml_to_audio
+
+import filecmp
+import os
+
+
+def test_text_to_ssml(capsys):
+    # Tests text_to_ssml() function
+    # Args: none
+    # Returns: none
+
+    # Assert plaintext converted to SSML
+    input_text = 'resources/example.txt'
+    tested_ssml = text_to_ssml(input_text)
+    expected_ssml = '<speak>123 Street Ln, Small Town, IL 12345 USA\n' \
```

Comment:
nit: You could read the lines from the `example.ssml` and compare to that and same for the `input_ssml` below so that there are fewer places that need to be changed for the future.